doc/install.md

```diff
@@ -3,8 +3,9 @@
 ## Table of Contents
 
 - [Create a Kubernetes Cluster](#create-a-kubernetes-cluster)
-- [Install release v0.19.8 and use Coscheduling](#install-release-v0198-and-use-coscheduling)
-  - [Test Coscheduling](#test-coscheduling)
+- [Install release v0.19.8 as a second scheduler and use Coscheduling](#install-release-v0198-as-a-second-scheduler-and-use-coscheduling)
```

Comment:
This may looks better:

```
- Install release v0.19.8 and use Coscheduling
    - as a single scheduler (replacing the vanilla default-scheduler)
    - as a secondary scheduler
```